src/libraries/Microsoft.Extensions.Logging.Console/ref/Microsoft.Extensions.Logging.Console.cs

```diff
@@ -11,10 +11,22 @@ public static partial class ConsoleLoggerExtensions
     {
         public static Microsoft.Extensions.Logging.ILoggingBuilder AddConsole(this Microsoft.Extensions.Logging.ILoggingBuilder builder) { throw null; }
         public static Microsoft.Extensions.Logging.ILoggingBuilder AddConsole(this Microsoft.Extensions.Logging.ILoggingBuilder builder, System.Action<Microsoft.Extensions.Logging.Console.ConsoleLoggerOptions> configure) { throw null; }
+        public static Microsoft.Extensions.Logging.ILoggingBuilder AddConsoleLogFormatter<TFormatter, TOptions>(this Microsoft.Extensions.Logging.ILoggingBuilder builder) where TFormatter : class, Microsoft.Extensions.Logging.Console.IConsoleLogFormatter where TOptions : Microsoft.Extensions.Logging.Console.SystemdConsoleLogFormatterOptions { throw null; }
+        public static Microsoft.Extensions.Logging.ILoggingBuilder AddConsoleLogFormatter<TFormatter, TOptions>(this Microsoft.Extensions.Logging.ILoggingBuilder builder, System.Action<TOptions> configure) where TFormatter : class, Microsoft.Extensions.Logging.Console.IConsoleLogFormatter where TOptions : Microsoft.Extensions.Logging.Console.SystemdConsoleLogFormatterOptions { throw null; }
+        public static Microsoft.Extensions.Logging.ILoggingBuilder AddDefaultConsoleLogFormatter(this Microsoft.Extensions.Logging.ILoggingBuilder builder, System.Action<Microsoft.Extensions.Logging.Console.DefaultConsoleLogFormatterOptions> configure) { throw null; }
+        public static Microsoft.Extensions.Logging.ILoggingBuilder AddJsonConsoleLogFormatter(this Microsoft.Extensions.Logging.ILoggingBuilder builder, System.Action<Microsoft.Extensions.Logging.Console.JsonConsoleLogFormatterOptions> configure) { throw null; }
+        public static Microsoft.Extensions.Logging.ILoggingBuilder AddSystemdConsoleLogFormatter(this Microsoft.Extensions.Logging.ILoggingBuilder builder, System.Action<Microsoft.Extensions.Logging.Console.SystemdConsoleLogFormatterOptions> configure) { throw null; }
     }
 }
 namespace Microsoft.Extensions.Logging.Console
 {
+    public static partial class ConsoleLogFormatterNames
+    {
+        public const string Default = "default";
+        public const string Json = "json";
+        public const string Systemd = "systemd";
+    }
+    [System.ObsoleteAttribute("ConsoleLoggerFormat has been deprecated.", false)]
```

Comment:
>ObsoleteAttribute [](start = 12, length = 17)

Do we need to file and fill the breaking change template for such changes?

Reply:
I'd say we don't. @PriyaPurkayastha?

Reply:
We actually do document APIs marked obsolete as a breaking change using https://github.com/dotnet/docs/issues/new?template=dotnet-breaking-change.md. 
You can look up https://docs.microsoft.com/en-us/dotnet/core/compatibility/aspnetcore for examples.

Reply:
- [ ] TODO

Reply:
/cc: @shirhatti